### PR TITLE
docs(ja): periodic update for v1.16

### PR DESCRIPTION
**Pull request by isleys**
periodic update for docs(ja)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v116.ja.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+このリリースに貢献した 7 人の新しいコミュニティ開発者を歓迎します。[@codekitchen](https://github.com/codekitchen),
+[@shingos](https://github.com/shingos) 、 [@csantos](https://github.com/csantos) 、 [@rfma23](https://github.com/rfma23)、
+[@g-grass](https://github.com/g-grass) 、 [@isleys](https://github.com/isleys) 、
+[@kangere](https://github.com/kangere) 。 [パブリックコミュニティチャット](https://gitter.im/aws/copilot-cli) は成長していて、270 人以上の方がオンラインで、日々助け合ってます。また、 最近 [GitHub](http://github.com/aws/copilot-cli/) で 2.1 K を超える Star を獲得しました。
+AWS Copilot に関心とサポートを示してくださった皆様に感謝しています。
+
+
+Copilot v1.16 にはいくつかの新機能と改善が加えられました:
+
+* **マルチパイプライン:** `copilot pipeline init` は、リポジトリ内の別々のブランチをつ席する複数の AWS CodePipeline を作成します。詳細は、
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: typo: つ席 to 追跡

File: site/content/blogs/release-v116.ja.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+[こちら](./#create-multiple-pipelines-per-branch)を確認してください。
+* **SNS サブスクリプションフィルターポリシー:** `filter_policy` フィールドを利用して、 Worker Service は、サブスクライブしたトピックに対する SNS メッセージをフィルターできます。
+詳細は、[こちら](./#define-messages-filter-policy-in-publishsubscribe-service-architecture)を確認してください。
+* **その他 たくさんの改善点:**
+    * `deploy` コマンドに `--no-rollback` フラグが追加されました。デプロイが失敗した場合にスタックの自動ロールバックを無効化します。([#3341](https://github.com/aws/copilot-cli/pull/3341)). 新しいフラグは、インフラストラクチャの更新失敗時にトラブルシューティングをする際に役立ちます。例えば、デプロイが失敗した場合、CloudFormation はスタックをロールバックするため、失敗時に発生したログを削除します。このフラグは、問題のトラブルシューティングの為にログを保存するのに役立ちます。保存後にロールバックし、マニフェストを再度更新できます。
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: I think the period of the sentence "([#3341](https://github.com/aws/copilot-cli/pull/3341))." should be Japanese one ("。").

File: site/content/blogs/release-v116.ja.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    * `package` コマンドに `--upload-assets` フラグが追加されました。CloudFormation テンプレートを生成する前に、ECR または S3 にアセットをプッシュします ([#3268](https://github.com/aws/copilot-cli/pull/3268))。 このフラグによりパイプラインの buildspec を大幅に簡略化ができます。 buildspec を再作成する場合は、ファイルを削除し、`copilot pipeline init` を再度実行します。その際に、プロンプトでは、既存のパイプライン名を指定します。
+    * Environment で `task run` 実行中にセキュリティグループを追加できるようになりました ([#3365](https://github.com/aws/copilot-cli/pull/3365))。
+    * Copilot が CI 環境で実行されている場合(環境変数 `CI` を `true` を設定している場合)に、Docker 進行状況の更新ノイズが少なくなりました ([#3345](https://github.com/aws/copilot-cli/pull/3345))。
+    * App Runner がサポートされていないリージョンに対して、App Runner Service をデプロイするときに警告ログを出力するようになりました([#3326](https://github.com/aws/copilot-cli/pull/3326))。
+    * `app show` および `env show` コマンドは、テーブル形式で Service や Job がデプロイされた Environment を出力するようになりました。([#3379](https://github.com/aws/copilot-cli/pull/3379)、[#3316](https://github.com/aws/copilot-cli/pull/3316))。
+    * パイプラインマニフェストの `build.buildspec` を使って buildspec パスをカスタマイズできるようになりました。([#3403](https://github.com/aws/copilot-cli/pull/3403)).
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: I think the period of the sentence "(https://github.com/aws/copilot-cli/pull/3403)." should be Japanese one ("。").

File: site/content/blogs/release-v116.ja.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+## AWS Copilot とは?
+
+AWS Copilot CLI は AWS 上でプロダクションレディなコンテナ化されたアプリケーションのビルド、リリース、そして運用のためのツールです。
+開発のスタートからステージング環境へのプッシュ、本番環境へのリリースまで、Copilot はアプリケーション開発ライフサイクル全体の管理を容易にします。
+Copilot の基礎となるのは、 AWS CloudFromation です。CloudFormation により、インプラストラクチャを 1 回の操作でコードとしてプロビジョニングできます。
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: typo: AWS CloudFromation to AWS CloudFormation.

File: site/content/blogs/release-v116.ja.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+_Contributed by [Efe Karakus](https://github.com/efekarakus/), [Janice Huang](https://github.com/huanjani/)_
+
+リリースプロセスを自動化することは、ソフトウエアデリバリの最も重要な部分の 1 つです。 AWS Copilot はそのプロセスをできるだけ簡単に設定できる様にしたいと考えています。
+アプリケーションの全ての Enviroment で `copilot deploy` を手動で実行する代わりに、
+いくつかの `copilot pipeline` コマンドを実行するだけで、 `git push` するたびに環境に対して自動的にリリースされる継続的デリバリーパイプラインをセットアップできます。
+
+生成された CodePipeline は、次の様な基本構造になります。
+
+* Source ステージ: 設定済みの GitHub、Bitbucket、または CodeCommit のリポジトリーのブランチに対して push した際に、新しいパイプライン実行がトリガーされます。
+* Build: リポジトリからコードがダウンロードされると、Service 用のコンテナイメージがビルドされ、すべての Environment の Amazon ECR リポジトリにプッシュされます。加えて、Adonn テンプレート、Lambda 関数 zip ファイル、環境変数ファイルなどのすべての入力ファイルが S3　にアップロードされます。
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: typo: Adonn テンプレート to Addon テンプレート

File: site/content/blogs/release-v116.ja.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+* Deploy ステージ: コードがビルドされた後、オプションのデプロイメント後テストや手動承認を仕様して、任意または全ての Environment にデプロイできます。
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: typo: 仕様 to 使用
I also think we should emphasize that post-deployment tests or manual approvals are optional. How about "コードがビルドされた後任意または全ての Environment にデプロイでき、オプションでデプロイ後のテストや手動承認を使用できます。"

File: site/content/blogs/release-v116.ja.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+    repository: https://github.com/user/repo
+stages:
+    - name: test
+    - name: prod
+    # requires_approval: true
+    # test_commands: [echo 'running tests', make test]
+```
+
+このモデルは "main" ブランチへの全てのコミットを Enviroment 全体でリリースしたいユーザに適しています。
+代替モデルは、ブランチごとにパイプラインを作成することです。例えば、"main" ブランチにいくつかの変更をコミットし、満足したら "test" ブランチに変更をマージして "test" Environment にデプロイします。次に "prod" ブランチにマージします。v1.16 までは単一のパイプラインマニフェストのみがサポートされていたため、このモデルは実現できませんでした。
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: The original sentence is "Until v1.16, this model was not possible ~" and customers would think Copilot v1.16 cannot create multiple pipelines. How about "v1.15 までは"?

File: site/content/blogs/release-v116.ja.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+
+v1.16 からは、Copilot ユーザは git リポジトリに複数のパイプラインを作成できる様になったので、ブランチごとに別々のパイプラインを設定できます。例えば、 マージ時のコンフリクトを気にすることなく、`copilot pipeline init を git リポジトリの別々のブランチで実行できます。
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: typo: "`copilot pipeline init を git リポジトリの別々のブランチで実行できます。" to "`copilot pipeline init` を git リポジトリの別々のブランチで実行できます。"

File: site/content/blogs/release-v116.ja.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+
+このフィルターポリシーを設定すると、Amazon SNS はこれらの属性とのマッチングにより全てのメッセージをフィルターします。
+SNS フィルターについてより学びたい場合は、[Copilot のドキュメント](../docs/manifest/worker-service.ja.md#topic-filter-policy)を確認してください。
+
+## 次は？
+
+以下のリンクをたどって、新しい Copilot CLI バージョンをダウンロードし、[GitHub](https://github.com/aws/copilot-cli/) や [コミュニティチャット](https://gitter.im/aws/copilot-cli)に
+フィードバックを残してください。
+
+* Download [the latest CLI version](../docs/getting-started/install.en.md)
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: Why not translate these three sentences?

File: site/content/blogs/release-v116.ja.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+
+✔ Wrote the pipeline manifest for repo-prod at 'copilot/pipelines/repo-prod/manifest.yml'
+Required follow-up actions:
+- Commit and push the copilot/ directory to your repository.
+- Run `copilot pipeline deploy` to create your pipeline.
+```
+
+"test" ブランチに対して変更がマージされると、 "repo-test" パイプラインがトリガーされます。同様に、"prod"ブランチに対して変更を反映すると "repo-prod" パイプラインをトリガーできます。
+
+パイプラインについてより詳しく学びたい場合は、[Copilot's docs](../docs/concepts/pipelines.ja.md)を確認してください。
```
Comment (hkford) [inline site/content/blogs/release-v116.ja.md]: Why not translate the phrase "Copilot's docs" like "Copilot のドキュメント"?

File: site/content/docs/concepts/pipelines.ja.md [hunk 11]
```diff
@@ -22,7 +22,10 @@ CodePipeline のような自動化されたリリースツールは、あなた
 Copilot はいくつかのコマンドで CodePipeline をセットアップします。作業を始める前に、まずは最終的に作成される Pipeline の構成を見ていきましょう。Pipeline は次に挙げる基本的な構造を持っています。
 
 1. __Source Stage__ - 設定した GitHub、Bitbucket、あるいは CodeCommit リポジトリにプッシュすると、Pipeline の実行が開始されます。
-2. __Build Stage__ - リポジトリからコードがダウンロードされると、Service 用のコンテナイメージがビルドされ、すべての Environment の Amazon ECR リポジトリにプッシュされます。
+2. __Build Stage__ - リポジトリからコードがダウンロードされると、Service 用のコンテナイメージがビルドされ、すべての Environment の Amazon ECR リポジトリにプッシュされます。加えて、[addons](../developing/additional-aws-resources.ja.md) テンプレートや、 Lambda 関数 zip ファイル、[環境変数ファイル]((../developing/environment-variables.ja.md))などのすべての入力ファイルが S3 にアップロードされます。
```
Comment (hkford) [inline site/content/docs/concepts/pipelines.ja.md]: I would like to capitalize the word 'addons'.

File: site/content/docs/concepts/pipelines.ja.md [hunk 12]
```diff
@@ -22,7 +22,10 @@ CodePipeline のような自動化されたリリースツールは、あなた
...
+
+ソースコードがリポジトリーホストから pull された後に、 Service のコンテナイメージがビルドされ、 各環境の ECR レポジトリーにパブリッシュされます。加えて、アドオンテンプレート、Lambda 関数 zip ファイル、環境変数ファイルなどのすべての入力ファイルが S3　にアップロードされます。
```
Comment (hkford) [inline site/content/docs/concepts/pipelines.ja.md]: typo: レポジトリー to リポジトリ

File: site/content/docs/manifest/worker-service.ja.md [hunk 13]
```diff
@@ -75,32 +79,58 @@ subscribe:
...
+  cutomer_interests:
+    - rugby
+    - football
+    - baseball
+  price_usd:
+    - numeric:
+      - ">="
+      - 100
+```
+さらにフィルターポリシーの書き方に関するさらに詳しい情報については、[SNS documentation](https://docs.aws.amazon.com/sns/latest/dg/sns-subscription-filter-policies.html)を確認してください。
```
Comment (hkford) [inline site/content/docs/manifest/worker-service.ja.md]: Duplicated word "さらに"

**Review by hkford — CHANGES_REQUESTED** [review]
Add some review comments.

File: site/content/blogs/release-v116.ja.md (hunk 1 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: fixed it

File: site/content/blogs/release-v116.ja.md (hunk 2 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: I agree with your proposal.  I fixed it.

File: site/content/blogs/release-v116.ja.md (hunk 3 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: Yes, I fixed it.

File: site/content/blogs/release-v116.ja.md (hunk 4 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: I fixed it.

File: site/content/blogs/release-v116.ja.md (hunk 5 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: I fixed it.

File: site/content/blogs/release-v116.ja.md (hunk 6 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: Your proposal is good. I adopt your idea.

File: site/content/blogs/release-v116.ja.md (hunk 7 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: OK. your idea is easier to understand.

File: site/content/blogs/release-v116.ja.md (hunk 8 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: fixed it.

File: site/content/blogs/release-v116.ja.md (hunk 9 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: I missed it.  I will fix.

File: site/content/blogs/release-v116.ja.md (hunk 10 above)
Comment (isleys) [reply site/content/blogs/release-v116.ja.md]: I missed it. I will fix.

File: site/content/docs/concepts/pipelines.ja.md (hunk 11 above)
Comment (isleys) [reply site/content/docs/concepts/pipelines.ja.md]: addons -> Addon

File: site/content/docs/concepts/pipelines.ja.md (hunk 12 above)
Comment (isleys) [reply site/content/docs/concepts/pipelines.ja.md]: レポジトリー to リポジトリ
アドオンテンプレート to Addon テンプレート

File: site/content/docs/manifest/worker-service.ja.md (hunk 13 above)
Comment (isleys) [reply site/content/docs/manifest/worker-service.ja.md]: Removed 1st words.

**Review by hkford — APPROVED** [review]
LGTM! Thank you @isleys for contributing!

**Review by efekarakus — APPROVED** [review]
Thank you so much @isleys and @hkford ! <3

**Pull request merged**